canonical-web-and-design/ubuntu.com
# Remove search temporarily

We are having issues with bad actors abusing the search endpoints so we are disabling search until we find a better solution to minimize the issue

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Make sure the search bar is gone
- Make sure /search is a 404
- Make sure Make sure /search?q=test is a 404
